semtype.py

```diff
@@ -0,0 +1,150 @@
+from dataclasses import dataclass, field
+
+class SyntacticFeatures:
```

Comment:
Reorder so that semtype is at the top and more specific content is at the bottom.